dashboard/config/linux/upstream-apparmor-kasan.config

```diff
@@ -509,13 +511,22 @@ CONFIG_CC_HAS_RETURN_THUNK=y
 CONFIG_CC_HAS_ENTRY_PADDING=y
 CONFIG_FUNCTION_PADDING_CFI=11
 CONFIG_FUNCTION_PADDING_BYTES=16
+CONFIG_CALL_PADDING=y
+CONFIG_HAVE_CALL_THUNKS=y
+CONFIG_CALL_THUNKS=y
+CONFIG_PREFIX_SYMBOLS=y
 CONFIG_SPECULATION_MITIGATIONS=y
-# CONFIG_PAGE_TABLE_ISOLATION is not set
-# CONFIG_RETPOLINE is not set
-CONFIG_CPU_IBPB_ENTRY=y
-CONFIG_CPU_IBRS_ENTRY=y
-# CONFIG_SLS is not set
-# CONFIG_GDS_FORCE_MITIGATION is not set
+CONFIG_MITIGATION_PAGE_TABLE_ISOLATION=y
+CONFIG_MITIGATION_RETPOLINE=y
+CONFIG_MITIGATION_RETHUNK=y
+CONFIG_MITIGATION_UNRET_ENTRY=y
+CONFIG_MITIGATION_CALL_DEPTH_TRACKING=y
+# CONFIG_CALL_THUNKS_DEBUG is not set
```

Comment:
Is this bug detection config?

Reply:
It's not a new config:
https://github.com/torvalds/linux/blob/861c0981648f5b64c86fd028ee622096eb7af05a/arch/x86/Kconfig#L2552

The description warns about `a noisy dmesg` and says `Only enable this when you are debugging call thunks as this creates a noticeable runtime overhead.